packages/hoppscotch-sh-admin/src/composables/usePagedQuery.ts

```diff
@@ -13,26 +13,29 @@ export function usePagedQuery<
   itemsPerPage: number,
   variables: Vars
 ) {
-  //Fetch All Users
   const { client } = useClientHandle();
   const fetching = ref(true);
   const error = ref(false);
   const list = ref<ListItem[]>([]);
   const currentPage = ref(0);
   const hasNextPage = ref(true);
 
-  const fetchNextPage = async () => {
+  const fetchNextPage = async (email?: string) => {
```

Comment:
Why is email hardcoded here ? Isn't this composable supposed to be more general ?